src/containers/EventContainer/EventContainer.tsx

```diff
@@ -103,8 +103,8 @@ const EventContainer: FC<EventContainerProps> = ({
 
   //Create the TranscriptItemsRefs to create jumpToTranscript callback
   const transcriptItemsRefs = useMemo(() => {
```

Comment:
Is the use of `useMemo` here so that if a component goes off screen and then returns, the cached version is used?

Reply:
If the value in the dependency array doesn't change, then it uses the cached version. This avoids re-calculation of the refs if it isn't needed (because the dependency array didn't change).

If the component gets destroyed then the value will have to be re-caclculated.